### PR TITLE
Create mumtazcs

### PR DESCRIPTION
**Pull request by Mumtaz12**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4138"></a>

### CODE REVIEW

**Comment by Mumtaz12** [comment]
thank you so much

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]
Hey, @Mumtaz12 You need to change your file name here, the file name must match your GitHub username, make it `data/Mumtaz12.json`
`.json` is also necessary

File: data/mumtazcs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    },
+   {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/mohammad-mumtaz-3957b4260/",
+      "icon": "FaLinkedIn"
+    },
+    {
+      "name": "Follow me on Youtube",
+      "url": "https://www.youtube.com/@DeepFakevolution",
+      "icon": "FaYouTube"
```
Comment (Aadarsh805) [inline data/mumtazcs:28]: this icon is wrong, it should be `FaYoutube`

File: data/mumtazcs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/withmumtazcs/",
+      "icon": "FaInstagram"
+    },
+   {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/mohammad-mumtaz-3957b4260/",
+      "icon": "FaLinkedIn"
```
Comment (Aadarsh805) [inline data/mumtazcs:23]: this icon should be `FaLinkedin`

File: data/mumtazcs [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    },
+   {
+      "name": "Follow me on Medium",
+      "url": "https://www.medium.com/IAmMumtaz",
+      "icon": "FaMedium"
+    },
+   {
+      "name": "Follow me on Hashnode",
+      "url": "https://hashnode.com/@Mumtaz",
+      "icon": "FaHashnode"
```
Comment (Aadarsh805) [inline data/mumtazcs:38]: and this should be `SiHashnode`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/mumtazcs (hunk 1 above)
Comment (Mumtaz12) [reply data/mumtazcs]: resolved

**Review by Mumtaz12 — COMMENTED** [review]
i updated

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Please note that your filename should be exact same as your username. 

So update if from `mumtazcs.json`  - >  `Mumtaz12.json`.

**Comment by Mumtaz12** [comment]
i changed my file name

File: data/Mumtaz12.json [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on Youtube",
+      "url": "https://www.youtube.com/@DeepFakevolution",
+      "icon": "FaYoutube"
+    },
+   {
+      "name": "Follow me on Medium",
+      "url": "https://www.medium.com/IAmMumtaz",
```
Comment (kumarsonsoff3) [inline data/Mumtaz12.json:32]: There is a broken URL. Please update it with the correct one. 🛠

**Review by kumarsonsoff3 — COMMENTED** [review]
Rest looks good to me! :+1:

**Comment by Mumtaz12** [comment]
where i will get link of linkfree profile

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good to merge. Thanks for your contribution! 🎉

<p align="center"></p>

**Comment by kumarsonsoff3** [comment]
> where i will get link of linkfree profile

This will be your linkfree profile link (http://linkfree.eddiehub.io/Mumtaz12). Your profile will be live once merged & deployed.

**Pull request merged**